### PR TITLE
Simulation update for demo

### PR DESCRIPTION
**Pull request by mscharfstein**
Works on #173 

Allows request date to be some time in the past to enable easier simulation of reasonable wait times (as opposed to everything being handled after 1 minute, or having to wait a while after adding the reservation to then complete it).

Also refactor the producer script slightly to match the consumer script and have clearer documentation. Remove the speed up factor as it's not currently handled

- [ ] Tests pass
- [ ] Appropriate changes to README are included in PR
- [ ] Feature has been fleshed out in design document

### CODE REVIEW

File: simulation/producer/main.js [hunk 1]
```diff
@@ -16,35 +15,43 @@ const argv = yargs
...
-    setInterval(sendReservations, delay)
+}
+
+/**
+ * Producer script for generating call requests. To ensure that the database does not
+ * receive a massive backlog of unaddressed calls, make sure that the --calls and --period
+ * parameters will introduce the same volume of calls that can be handled by the script
+ * at ../consumer/main.js.
+ * 
+ * sample call: node main.js --calls 100 --period 60 --speed-up 120 --api-url 'http://192.168.64.4:31442'
```
Comment (kebrahim) [inline simulation/producer/main.js:51]: remove --speed-up and add in the new rewind param

File: simulation/producer/lib/simulation.js [hunk 2]
```diff
@@ -2,7 +2,7 @@ const axios = require('axios');
 const faker = require('faker');
 const moment = require('moment');
 
-function sendReservation(apiUrl) {
+function sendReservation(apiUrl, rewindRequestDateMins) {
```
Comment (kebrahim) [inline simulation/producer/lib/simulation.js:5]: while you're here, mind documenting this method?

Comment (mscharfstein) [reply simulation/producer/lib/simulation.js:13]: done

File: simulation/producer/main.js (hunk 1 above)
Comment (mscharfstein) [reply simulation/producer/main.js]: done

**Pull request merged**